exasol/analytics/query_handler/query/select.py

```diff
@@ -104,13 +135,15 @@ def __init__(
         db_operation_type: DbOperationType,
         audit_fields: dict[str, Any] | None = None,
         audit: bool = False,
+        parent_log_span: LogSpan | None = None,
     ):
         CustomQuery.__init__(self, query_string)
         AuditData.__init__(self, audit_fields)
         self._db_object_type = db_object_type
         self._db_object_name = db_object_name
         self._db_operation_type = db_operation_type
         self._audit = audit
+        self._log_span = LogSpan(db_operation_type.name, parent=parent_log_span)
```

Comment:
is there a reason, you don't use parent_log_span.child(...)

Reply:
OMG - you're right. I forgot about that. :confused:

Reply:
Now I know, why: `parent_log_span` is optional. 

The question is: Do we want to have a `LogSpan` for each `ModifyQuery`, even if `parent_log_span` is `None`?